justfile

```diff
@@ -36,3 +36,20 @@ forbidden-words:
     'dbg!\|fixme\|todo\|#\[ignore\]' \
     src tests examples
   @echo No forbidden words found
+
+all-rustc-versions *args="ci":
+  #!/usr/bin/env bash
+  set -eu
+
+  export RUSTFLAGS="--deny warnings"
+
+  # install yq with: pip3 install yq
+  versions=$(cat .github/workflows/ci.yaml \
+    | yq -r '.jobs.all.strategy.matrix.rust | sort | join(" ")' \
+    ;)
+  for version in $versions
+  do
+    rustup override set $version
```

Comment:
If the recipe fails, will the override still be set?

Reply:
Yes. I thought there was no easy way to avoid that, but then I found [`RUSTUP_TOOLCHAIN`](https://rust-lang.github.io/rustup/environment-variables.html). Good catch! Fixed in 3bfc22ad.

Reply:
Nice, LGTM